discovery-infra/test_infra/utils/utils.py

```diff
@@ -930,11 +908,12 @@ def download_iso(image_url, image_path):
 
 
 def parse_olm_operators_from_env():
```

Comment:
You should check, but I think most of the functions here are internals, and we don't need to keep the old function. Just move their references in case they are in this repository.

Reply:
`start_discovery` and ``conftest` are using this method.

Reply:
So change the calling functions as well

Reply:
Soon `start_discovery`, `conftest` and other scripts will be deprecated (or deleted) so changing those scripts is a bit duplicate job. Also some of those function are used by the QE and I don't want to be blocked by them (or scan their project for every change)